src/AddressRegistry.Api.BackOffice.Abstractions/Requests/AddressCorrectBoxNumberRequest.cs

```diff
@@ -0,0 +1,51 @@
+namespace AddressRegistry.Api.BackOffice.Abstractions.Requests
+{
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+    using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+    using MediatR;
+    using Newtonsoft.Json;
+    using Responses;
+    using StreetName;
+    using StreetName.Commands;
+    using Swashbuckle.AspNetCore.Filters;
+
+    [DataContract(Name = "CorrigerenBusnummerAdres", Namespace = "")]
```

Comment:
I thought the DataContract attribute should be removed at this level for requests where persistent local id is retrieved from URL?

Reply:
Contract is needed here, because of the boxnumber input